src/EPG.cpp

```diff
@@ -132,13 +129,65 @@ PVR_ERROR EPG::GetEPGForChannel(int channelUid, time_t start, time_t end, kodi::
         }
 
       }
-      broadcast.SetSeriesNumber(XMLUtils::GetIntValue(pListingNode, "season", EPG_TAG_INVALID_SERIES_EPISODE));
-      broadcast.SetEpisodeNumber(XMLUtils::GetIntValue(pListingNode, "episode", EPG_TAG_INVALID_SERIES_EPISODE));
+
+      int season{ EPG_TAG_INVALID_SERIES_EPISODE };
+      int episode = {EPG_TAG_INVALID_SERIES_EPISODE};
```

Comment:
Remove the `=`, it is not needed here.

Reply:
Yes that it is in Omega too (came from the port).  Fixed here not in Omega.